src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,24 @@
-import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
```

Comment:
Use styling for making font-weight bolder, it's bad practice of using tags. Maybe consider using headings as a semantic element for titles